query/query.go

```diff
@@ -197,9 +205,22 @@ func (q *RepoBranches) String() string {
 }
 
 // Branches returns a query representing the branches to search for name.
-func (q *RepoBranches) Branches(name string) Q {
-	branches, ok := q.Set[name]
-	if !ok {
+func (q *RepoBranches) Branches(name string, id uint32) Q {
+	var branches []string
+
+	switch {
```

Comment:
use if/else if

Reply:
My style would also prefer `if/else`, but not a blocker since it is stylistic.

Reply:
Let's then agree to use if else for two branches, and switch for 2+ branches.